data-plane/config/broker-tls/broker-ingress-tls-certificate.yaml

```diff
@@ -40,6 +40,7 @@ spec:
 
   dnsNames:
     - kafka-broker-ingress.knative-eventing.svc.cluster.local
+    - kafka-broker-ingress.knative-eventing.svc
```

Comment:
Can we do this for other certificates? Including the one in eventing core?

Reply:
@pierDipi  Added for channel and sink. Also created the new issue in eventing core. https://github.com/knative/eventing/issues/7209

Is this what you are suggesting by saying "Including the one in eventing core" ?